web/bg.lproj/Localizable.strings

```diff
@@ -16,28 +16,28 @@
 "ADDON_MADE_BY_AUTHOR_TITLE_TEMPLATE" = "От: %@";
 "COMMON_ERROR_TITLE" = "Грешка";
 
-"ERROR_NOT_FOUND" = "The requested content is not found.";
-"ERROR_FIELDS_MISSING" = "Some fields were missing in the request.";
-"ERROR_INCORRECT_PARAMETERS" = "Some parameters were incorrect.";
-"ERROR_INTERNAL_ERROR" = "Internal server error.";
+"ERROR_NOT_FOUND" = "Съдържанието не е намерено.";
+"ERROR_FIELDS_MISSING" = "Има липсващи полета.";
+"ERROR_INCORRECT_PARAMETERS" = "Има неточни параметри.";
+"ERROR_INTERNAL_ERROR" = "Вътрешна грешка на сървъра.";
 
-"ADDON_POPULAR_TITLE" = "Popular Add-ons";
+"ADDON_POPULAR_TITLE" = "Популярни добавки";
 
-"ADDON_INSTALL_HINT_NO_DEMO_OBJECT" = "Note: restarting Celestia is needed to use any new installed add-on. This add-on does not have a demo object, so only \"Uninstall\" option is visible, please navigate to the object by using search.";
+"ADDON_INSTALL_HINT_NO_DEMO_OBJECT" = "Забележка: Рестартирайте „Celestia“ за активиране на новите инсталирани добавки. Тази добавка няма демонстрационен обект, затова само опцията \„Премахни\“ е видима, трябва да използвате търсачката за да намерите обекта.";
```

Comment:
please remove the \ before „ and “. the English version uses `\"` because `"` needs to be escaped. not needed for `„` or `“`